sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1385,8 +1385,10 @@ abstract class DDLSuite extends QueryTest with DDLSuiteBase {
             exception = intercept[AnalysisException] {
               sql(s"SHOW COLUMNS IN $db.showcolumn FROM ${db.toUpperCase(Locale.ROOT)}")
             },
-            errorClass = "_LEGACY_ERROR_TEMP_1057",
-            parameters = Map("dbA" -> db.toUpperCase(Locale.ROOT), "dbB" -> db)
+            errorClass = "SHOW_COLUMNS_WITH_CONFLICT_NAMESPACE",
```

Comment:
shall we move all the related test cases to the new `ShowColumnsSuiteBase`?

Reply:
Okay, let me to do it.

Reply:
> shall we move all the related test cases to the new `ShowColumnsSuiteBase`?

Done.